spec.md

```diff
@@ -20,32 +20,28 @@ This Artifacts Spec provides a reference for artifact authors and registry imple
 
 ## Scope
 
-Create a first version of the Artifacts specification that is dependent on [OCI manifest][image-manifest] 1.0 to represent individual artifact types including [OCI Image][image-spec], [Helm][helm] and [Singularity][singularity].
-
-**v1.0 of the image-spec** defines index and manifest.
+Create a first version of Artifacts support which is dependent on [OCI manifest][image-manifest] 1.0 to represent individual artifact types including [OCI Image][image-spec], [Helm][helm] and [Singularity][singularity].
 
 <img src="./media/image-spec-objects.png" width=500x>
 
-**Artifacts 1.0** utilize manifest and index to represent multiple artifact types, including the [OCI Image-spec][image-spec]
-
 <img src="./media/artifacts-spec-objects.png" width=500x>
 
 ### Future Scope
 
-Future versions of this specification will support new artifact types representing collections of artifacts using [OCI Index][image-index]. A means to identify an index as a type of artifact will be required.
+Future versions will support new artifact types representing collections of artifacts using [OCI Index][image-index]. A means to identify an index as a type of artifact will be required.
 
 ## Defining OCI Artifact Types
 
 As registries become content addressable distribution points, tools that pull artifacts must know if they can operate on the artifact. Artifact types are equivalent to file extensions.
 
-As a set of comparable file extension use cases:
+A set of comparable file extension use cases include:
 
 - When users open files, the host operating system typically launches the appropriate program.
 - When users open a file, from within a program, the open dialog filters to the supported types.
 - When search or security software scan the contents of a storage solution, the software must know how to process the different types of content.
 - When users view the storage contents, information about the content is rendered based on the derived type(s) of the content being shown.
 
-This OCI Artifacts specification enables these use cases for [OCI distribution spec][distribution-spec] based registries by specifying a `manifest.config.mediaType` on the content pushed to a registry.
+OCI Artifacts enables these use cases for [OCI distribution spec][distribution-spec] based registries by specifying a `manifest.config.mediaType` on the content pushed to a registry.
```

Comment:
suggest adding a TODO () for examples or add a link to where these will be specified.

Reply:
The next PR will focus on publishing well-known types with specific examples for OCI, Helm and a document for publishing the well-known type information. We'll have a folder of examples. 
The example below shows the result of doing this work.
To keep each PR clean, I've removed the TODOs and kept focused on what's in each document. This way, each merge stands on it's own and is complete.

Reply:
The coming `publishing-well-known-types.md` PR will have several examples, including the `.json` file for describing a well-known type. To allow this PR to be cleanly merged, I removed TODOs.